### PR TITLE
Added instructions to create a RootCA and a client certificate and bind it to IIS web site

### LINKED ISSUE #680: Feedback for Enable and Set Up IIS
You should also explain in this section that they should setup https on 443 because when I tried to run the website as a developer it would not work for me using https://www.dnndev.me until I setup https on my local IIS for the website.

Thanks,

Scott

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.dnncommunity.org ➟ Core Team processing.*

* Content Source: [set-up-iis](https://github.com/DNNCommunity/DNNDocs/blob/main/content/tutorials/server/setup/set-up-iis/index.md/#L1)

### PR DESCRIPTION
**Pull request by MaiklT**
Closes #680

### CODE REVIEW

File: content/tutorials/extensions/dnn-manifest-schema/index.md
Comment (david-poindexter) [inline content/tutorials/extensions/dnn-manifest-schema/index.md:1]: I don't believe this file belongs in this PR.

**Review by david-poindexter — CHANGES_REQUESTED** [review]
@MaiklT perhaps this belongs in a blog post instead of documentation?  This is quite opinionated (not that it is wrong) in general and there are many ways to go about doing this based on personal preference.  What do you think?

File: content/tutorials/extensions/dnn-manifest-schema/index.md
Comment (MaiklT) [reply content/tutorials/extensions/dnn-manifest-schema/index.md:1]: You are right - something got mixed up.

**Review by MaiklT — COMMENTED** [review]
Please delete this pull request, thanks!

**Pull request closed without merging**